rotkehlchen/exchanges/iconomi.py

```diff
@@ -197,8 +198,8 @@ def validate_api_key(self) -> Tuple[bool, str]:
         except RemoteError:
             return False, 'Provided API Key is invalid'
 
-    def query_balances(self, **kwargs: Any) -> Tuple[Optional[Dict[Asset, Dict[str, Any]]], str]:
-        balances = {}
+    def query_balances(self, **kwargs: Any) -> ExchangeQueryBalances:
+        asset_balance: Dict[Asset, Balance] = {}
```

Comment:
Can you rename it to either assets_balance or asset_balances or anything that indicates it is plural?

Reply:
Ha! The classic map naming convention... singular? plural? To be honest for maps I like to compose the name with:
- key name: type of the key data (e.g. `asset`)
- value name: type of the value data (`e.g.`balance`)
- And separate them with double underscore (just in case the key or the value name has already an underscore).

A dict between <Asset> and <Balance> would be `asset__balance`. And a map between <Asset> and sequence of <Balance> would be `asset__balances`.
Don't worry that I'm not using this naming convention at Roki :P. Other people like to add the variable type/structure in the name (e.g. `asset_balance_map`) whilst others say that adding it is bad practice :S

Reply:
I'll go with `assets_balance` everywhere and even `assets_total_balance` in `rotkehlchen.py::query_balances`.

Reply:
Okay. In general let's try to get a convention and stick to it in the future.